spec/classes/docker_spec.rb

```diff
@@ -422,7 +422,6 @@
       it { should compile.with_all_deps }
       it { should contain_class('docker::repos').that_comes_before('Class[docker::install]') }
       it { should contain_class('docker::install').that_comes_before('Class[docker::config]') }
-      it { should contain_class('docker::service').that_subscribes_to('Class[docker::config]') }
```

Comment:
Can we add in as part of this ensuring that the config class comes before the service class?